apcd-cms/src/apps/utils/apcd_database.py

```diff
@@ -406,35 +406,30 @@ def create_registration_entity(entity, reg_id, from_update_reg=None):#, old_reg_
             conn.close()
 
 
-def update_registration_entity(form, reg_id, iteration, no_entities):
+def update_registration_entity(entity, reg_id):
     cur = None
     conn = None
     values = ()
     try:
-        str_end = f'{iteration}_{reg_id}'
-        if not _acceptable_entity(form, iteration, reg_id):
-            if iteration <= no_entities: # entity is not in form but was in original entity list -> need to delete
-                return delete_registration_entity(reg_id, form['ent_id_{}'.format(str_end)])
-            return
-        if iteration > no_entities: # entity is in form but not in original list -> need to create
-            return create_registration_entity(form, reg_id, iteration, from_update_reg=True)
+        if entity['entity_id'] < 0:
```

Comment:
I need to handle the case where entity or contact is deleted. Will address that with a commit in this PR.

Reply:
Done handled